src/vm/ceemain.cpp

```diff
@@ -854,15 +855,15 @@ void EEStartupHelper(COINITIEE fFlags)
 #endif // !FEATURE_CORECLR && !CROSSGEN_COMPILE
         }
 
-#ifdef CROSSGEN_COMPILE
+#if defined(CROSSGEN_COMPILE) && defined(FEATURE_MERGE_JIT_AND_ENGINE)
 //ARM64TODO: Enable when jit is brought in
```

Comment:
You could remove this ARM64 TODO and ifdef while you are on it - it is no longer relevant.